x/crconsent/handler_request.go

```diff
@@ -0,0 +1,51 @@
+package crconsent
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+    sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/metabelarus/mbcorecr/x/crconsent/types"
+	"github.com/metabelarus/mbcorecr/x/crconsent/keeper"
+)
+
+func handleMsgCreateRequest(ctx sdk.Context, k keeper.Keeper, msg *types.MsgCreateRequest) (*sdk.Result, error) {
+	k.CreateRequest(ctx, *msg)
+
+	return &sdk.Result{Events: ctx.EventManager().ABCIEvents()}, nil
```

Comment:
Make sure that new record id is returned.
Look at x/crsign/handler_record.go:65